stable/mediawiki/templates/deployment.yaml

```diff
@@ -95,12 +105,12 @@ spec:
           httpGet:
             path: /index.php
             port: http
-          initialDelaySeconds: 120
+{{ toYaml .Values.livenessProbe | indent 10 }}
         readinessProbe:
```

Comment:
it should also be possible to completely disable the liveness/readiness probes

Reply:
👍